tests/methods/test_reft.py

```diff
@@ -0,0 +1,78 @@
+from adapters import DiReftConfig, LoReftConfig
+from transformers.testing_utils import require_torch
+
+from .base import AdapterMethodBaseTestMixin
+
+
+@require_torch
+class ReftTestMixin(AdapterMethodBaseTestMixin):
+    reft_configs_to_test = [
+        (LoReftConfig(), ["refts.{name}."]),
```

Comment:
Should we maybe include a config with a suffix as well? With the logic of finding the last token, I think this might be more prone to breaking than the prefix.

Reply:
yep, added